stacks-common/src/util/log.rs

```diff
@@ -238,7 +238,8 @@ fn make_logger() -> Logger {
         let plain = slog_term::PlainSyncDecorator::new(slog_term::TestStdoutWriter);
         let isatty = isatty(Stream::Stdout);
         let drain = TermFormat::new(plain, false, debug, isatty);
-        Logger::root(drain.ignore_res(), o!())
+        let logger = Logger::root(drain.ignore_res(), o!());
+        logger
```

Comment:
I think for this one, you can take the other version. It probably avoids a clippy warning.

Reply:
reverted

Reply:
on second thought, we may want to keep this and adjust clippy - i believe this is part of a hotfix: https://github.com/stacks-network/stacks-core/commit/34703d7b63135987c296014bff7377b4115ff57f

Reply:
The change from `fuse` to `ignore_res` is the important part for the hotfix. The simplification from defining a variable and then returning it is what clippy complains about.